packages/core/__test__/registries/ElementRegistry.test.js

```diff
@@ -104,6 +104,8 @@ describe('TestElementRegistry', function () {
   });
 
   it('Test define twice same name', async function () {
+    // eslint-disable-next-line no-console
+    console.info('hostname:', location.hostname);
```

Comment:
let's forget about this and update test case itself instead. Expect exceptions from localhost only + expect no exceptions from non-localhost hostname.